libass/ass_render.c

```diff
@@ -70,15 +71,15 @@ ASS_Renderer *ass_renderer_init(ASS_Library *library)
     priv->ftlibrary = ft;
     // images_root and related stuff is zero-filled in calloc
 
+    priv->engine = &ass_bitmap_engine_c;
+
+    ASS_CPUFlags flags = ass_get_cpu_flags();
```

Comment:
Causes `-Wunused-variable` when compiling without ASM.
I think an `#if CONFIG_ASM` around this and the architecture checks avoids this and imho works better for future architecture additions.